src/App.tsx

```diff
@@ -17,17 +17,15 @@ import './App.css';
 
 const App: FC = () => {
   const dispatch = useDispatch();
-  const signedIn = useSelector(authSelectors.selectAuthSignedIn);
-  const [routes, setRoutes] = useState(ROUTES.defaultRoutes);
+  const signedIn = useSelector(selectAuthSignedIn);
+  const [routes, setRoutes] = useState(defaultRoutes);
 
   useEffect(() => {
-    dispatch(authActions.checkAuth());
+    dispatch(checkAuth());
   }, [dispatch]);
 
   useEffect(() => {
-    signedIn
-      ? setRoutes([...ROUTES.authenticatedRoutes])
-      : setRoutes([...ROUTES.defaultRoutes]);
+    signedIn ? setRoutes(authenticatedRoutes) : setRoutes(defaultRoutes);
```

Comment:
```
setRoutes(signedIn
  ? authenticatedRoutes
  : defaultRoutes
)
```
Not important, but this would have worked as well

Reply:
haha! didn't thought of that small detail, makes it more compact, thanks!